### ISSUE TITLE
Programming Panel: Only color is used to show links - 1.4.1 Use of Color

### ISSUE THREAD

**Issue by syarwick23** [stephenravenscraft/loyola-template-4.0#103]
For the main calendar link and all of the links on "Feature Multiple Links", the only way to show the text is links are the color (Maroon vs black). 

<img width="879" height="564" alt="Image" src="https://github.com/user-attachments/assets/132a0d11-4e74-40a1-ae77-77a5ce969406" />
